FirstProject/new-evaluation.html

```diff
@@ -56,29 +57,29 @@ <h2 class="description_section">Technical level</h2>
         <div class="row_section">
             <div class="col-md-3 bullets_menu">
                 <form action="">
-                    <input type="radio" name="circle" value="circles">
+                    <input class="input-circle" type="radio" name="circle" value="circles">
```

Comment:
Curious... What does this piece of code supposed to achieve? Why is it repeated throughout?